xtras/src/supervisor.rs

```diff
@@ -356,15 +298,13 @@ mod tests {
         std::panic::set_hook(Box::new(|_| ())); // Override hook to avoid panic printing to log.
 
         let (supervisor, address) =
-            Actor::with_policy(|| PanickingActor, always_restart::<io::Error>());
-        let (supervisor, task) = supervisor.create(None).run();
+            Supervisor::with_policy(|| PanickingActor, always_restart::<io::Error>());
+        let task = supervisor.run();
 
-        #[allow(clippy::disallowed_methods)]
-        tokio::spawn(task);
-
-        address.send(Panic).await.unwrap_err(); // Actor will be dead by the end of the function call because it panicked.
```

Comment:
It is no longer true that the actor will be dead by the end of the `send` call, since in this case the `send` is split off, as I rewrote the test to not have a parallel-running supervisor, but run it on the same thread.